fmn/rules/taskotron.py

```diff
@@ -82,13 +82,13 @@ def taskotron_task_outcome(config, message, outcome=None):
 
 @hint(categories=['taskotron'], invertible=False)
 def taskotron_task_particular_or_changed_outcome(config, message,
-                                                 outcome='FAILED'):
+                                                 outcome='FAILED,NEEDS_INSPECTION'):
     """ Taskotron task any particular or changed outcome(s)
 
     With this rule, you can limit messages to only those task results
-    with any particular outcome(s) (FAILED by default) or those with
-    changed outcomes. This rule is a handy way of filtering a very
-    useful use case - being notified when either task failed for an
+    with any particular outcome(s) (FAILED and NEEDS_INSPECTION by default)
+    or those with changed outcomes. This rule is a handy way of filtering
+    a very useful use case - being notified when either task failed for an
```

Comment:
> when either task failed

Maybe "when either task requires your attention" (to cover both `FAILED` and `NEEDS_INSPECTION` cases)?